docs/README.md

```diff
@@ -249,6 +252,12 @@ Use the following URL in all HTTP requests that you send to the resource aggrega
 
 - {port} is the port where the services of the resource aggregator are running. The default port is 45000. If you
     have changed the default port in the `odim_config.json` file, use that as the port.
+	
+	
+To access the base URL using a REST client, use IP address of the system where the resource aggregator is installed as `{odim_host}`.
```

Comment:
may be you can write like- Replace `{odimra_host}` in the URL with IP address of the system where the resource aggregator is installed